x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/compliance_dashboard.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import Chance from 'chance';
+import type { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const retry = getService('retry');
+  const pageObjects = getPageObjects(['common', 'svlCommonPage', 'cloudPostureDashboard']);
+  const chance = new Chance();
+
+  const data = [
+    {
+      '@timestamp': new Date().toISOString(),
+      resource: { id: chance.guid(), name: `kubelet`, sub_type: 'lower case sub type' },
+      result: { evaluation: 'failed' },
+      rule: {
+        name: 'Upper case rule name',
+        section: 'Upper case section',
+        benchmark: {
+          id: 'cis_k8s',
+          posture_type: 'kspm',
+        },
+      },
+      cluster_id: 'Upper case cluster id',
+    },
+  ];
+
+  describe('Cloud Posture Dashboard Page', function () {
+    // TODO: we need to check if the tests are running on MKI. There is a suspicion that installing csp package via Kibana server args is not working on MKI.
+    this.tags(['skipMKI', 'cloud_security_posture_compliance_dashboard']);
```

Comment:
do we track that on an issue? if not, lets open a ticket for it

Reply:
yes, we track that in https://github.com/elastic/security-team/issues/7735